<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphTab</class>
 <widget class="QDialog" name="GraphTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1296</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QwtPlot" name="qwtPlot">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>1031</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SelectButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QComboBox" name="StockComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="removePlotButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>221</width>
     <height>251</height>
    </rect>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
